examples/use-cases/Vfio2Noop/README.md

```diff
@@ -50,9 +50,27 @@ spec:
 EOF
 ```
 
+Create kustomization file:
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ns-vfio2noop
+
+resources:
+- https://github.com/networkservicemesh/deployments-k8s/apps/nsc-vfio?ref=2dbc67a434ce864fe0aca418e578d2fa78e2fcb6
+- https://github.com/networkservicemesh/deployments-k8s/apps/nse-vfio?ref=2dbc67a434ce864fe0aca418e578d2fa78e2fcb6
+
+patchesStrategicMerge:
+- patch-nse-vfio.yaml
+EOF
+```
```

Comment:
Seems like can be a static file.

Also, links can be changed to relative paths.

Reply:
Fixed